pages/common/puppet-agent.md

```diff
@@ -22,3 +22,11 @@
 - Log every resource being evaluated (even if nothing is being changed):
 
 `puppet agent --test --evaltrace`
+
+- Disable puppet agent
+
+`puppet agent --disable "<your message>"`
```

Comment:
We try to avoid `you/your`. We use this token syntax and `snake_case` for placeholders.
```suggestion
`puppet agent --disable "{{message}}"`
```